### PR TITLE
fix: remove the dup snackbar

### PR DESCRIPTION
**Pull request by guanbinrui**
closes #

### CODE REVIEW

File: packages/mask/src/plugins/ITO/SNSAdaptor/ClaimAllDialog.tsx [hunk 1]
```diff
@@ -210,10 +210,7 @@ const useStyles = makeStyles<StyleProps>()((theme, props) => ({
         width: 535,
         margin: '24px auto',
     },
-    claimAllButton: {
-        background: `${theme.palette.mode === 'light' ? '#111418' : '#EFF3F4'} !important`,
-        color: `${theme.palette.mode === 'light' ? '#FFFFFF' : '#0F1419'} !important`,
```
Comment (zhouhanseng) [inline packages/mask/src/plugins/ITO/SNSAdaptor/ClaimAllDialog.tsx:215]: Please revert this change since switch button color fixed at #4955

Comment (guanbinrui) [reply packages/mask/src/plugins/ITO/SNSAdaptor/ClaimAllDialog.tsx:215]: Will do the Twitter button theme following in another PR. The dead code button color is not a good idea.
